Source/WebKit/UIProcess/WebAuthentication/fido/CtapAuthenticator.cpp

```diff
@@ -172,6 +179,7 @@ void CtapAuthenticator::continueMakeCredentialAfterResponseReceived(Vector<uint8
 
 void CtapAuthenticator::getAssertion()
 {
+    CTAP_RELEASE_LOG("getAssertion");
```

Comment:
Is this necessary? The logging below will always be called, so this seems redundant.